README.md

```diff
@@ -1,7 +1,78 @@
 <!---<img src="docs/markdown/images/Header.png" alt="drawing" width="1000"/>
 
 # GridPACK<sup>TM</sup>-->
-# GridPACK: High-Performance Electric Grid Simulation
+
+GridPACK is a software framework to simplify the development of programs that model the power grid to run on parallel, high performance computing platforms. GridPACK comes with a set of prebuilt applications (programs) that can be readily used. The
```

Comment:
There is already a GridPACK overview description below. So what is the need for this description?

Reply:
Okay, you might want to trim the first sentence in the description below, but it is probably a good idea if you have a quick summary at the very top of the documentation.